dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/CompanyChecks.java

```diff
@@ -0,0 +1,7 @@
+package val31.web;
+
+/**
+*
+*/
+public interface CompanyChecks {
```

Comment:
This interface isn't necessary and can be removed

Reply:
Updated